File: tests/go/cases/case_219_dead_sink_decoy_02.go
package escape_tests

var retainedCase219 = []map[string]string{}

func Case219DeadSinkDecoy02(input string) string {
	raw := input
	if raw == "" {
		raw = "sample"
	}
	payload := map[string]string{
		"task": "dead_sink_decoy_02",
		"entity": "extreme",
		"stage": "stress",
		"input": raw,
		"checksum": "dead_sink_decoy_02:" + raw,
	}
	if 1 == 0 {
		retainedCase219 = append(retainedCase219, payload)
	}
	// SAFE: dead branch never executes sink.
	return payload["checksum"]
}

File: tests/nodejs/cases/case_219_dead_sink_decoy_02.js
/**
 * Task case 219: dead_sink_decoy_02 deep stress pattern.
 */

const retainedCache = new Map();
const retainedAudit = [];
const retainedHandlers = [];

function case219DeadSinkDecoy02(input) {
  const taskName = 'dead_sink_decoy_02';
  const raw = input || 'sample';
  const payload = {
    task: taskName,
    entity: 'extreme',
    stage: 'stress',
    input: raw,
    checksum: `${taskName}:${raw.length}`,
  };
  if (1 === 2) { retainedAudit.push(payload); }
  // SAFE: dead branch only.
  return payload.checksum;
}

module.exports = { case219DeadSinkDecoy02 };
